src/Avatar.js

```diff
@@ -67,7 +67,7 @@ module.exports = {
      * @return {string} the first letter
      */
     getInitialLetter(name) {
-        if (name.length < 1) {
+        if (!name || name.length < 1) {
```

Comment:
a `console.trace("undefined name")` might be worthwhile. Would be good if/when we ever get an automatic bug reports option.